### PR TITLE
v0.30: Update telemetry table

### LINKED ISSUE #1971: v0.30: Update telemetry table
v0.30 introduces at least one new data point. We need to update the telemetry table in the "Telemetry".

## Needs further clarification
Due to the new process for feature implementation, not all new features have a written spec for reference. We need to coordinate with @gmourier to ensure we don't miss anything.

## Tasks
- [ ] Add new env vars to existing options
  - `--import-dump -> MEILI_IMPORT_DUMP`
  - `--ignore-missing-dump -> MEILI_IGNORE_MISSING_DUMP`
  - `--ignore-dump-if-db-exists -> MEILI_IGNORE_DUMP_IF_DB_EXISTS`
  - `--import-snapshot -> MEILI_IMPORT_SNAPSHOT`
  - `--ignore-missing-snapshot -> MEILI_IGNORE_MISSING_SNAPSHOT`
  - `--ignore-snapshot-if-db-exists -> MEILI_IGNORE_SNAPSHOT_IF_DB_EXISTS`

## References
- SME: @gmourier

### PR DESCRIPTION
**Pull request by maryamsulemani97**
Closes #1971

### CODE REVIEW

**Comment by gmourier** [comment]
Hey @maryamsulemani97, thanks for the pull request.

We've just added a last-minute change that you may want to mention here. Since `Tasks Seen` is now an aggregated event, we added a `requests.total_received` property.

The specification also mentioned that property for the newly added `Documents Deleted` and `Health Seen` events.

**Review by gmourier — COMMENTED** [review]
It seems that this [part](https://github.com/meilisearch/specifications/pull/186/files#r1030499496) is missing here

File: learn/what_is_meilisearch/telemetry.md [hunk 1]
```diff
@@ -144,32 +144,59 @@ This list is liable to change with every new version of Meilisearch. It's not be
...
-| `most_used_matching_strategy`                      | The most used word matching strategy among all search requests in the aggregated event      | `last`
+| `matching_strategy.most_used_strategy`             | Most used word matching strategy                                                            | last
 | `infos.disable_auto_batching`                      | `true` if `--disable-auto-batching`/`MEILI_DISABLE_AUTO_BATCHING` is specified, otherwise `false` | true
+| `infos.with_configuration_file`                    | `true` if the instance is launched with a configuration file, otherwise `false`             | false
+| `swap_operation_number`                            | Number of swap operations                                                                   | 2
+| `pagination.most_used_navigation`                  | Most used search results navigation                                                         | estimated
+| `requests.total_received`                          | Total number of requests received                                                           | 3480
```
Comment (gmourier) [inline learn/what_is_meilisearch/telemetry.md:202]:
```suggestion
```

This one is already in the table 😇

**Review by gmourier — COMMENTED** [review]
It seems that [part](https://github.com/meilisearch/specifications/pull/186/files#r1030512119) is missing.

**Review by dichotommy — APPROVED** [review]
Onward! ⏭️

**Comment by maryamsulemani97** [comment]
bors merge 🚂

**Pull request merged**